app/recipes/views.py

```diff
@@ -0,0 +1,39 @@
+# from rest_framework.decorators import action
+# from rest_framework.response import Response
+from rest_framework import viewsets, mixins
+from core.models import Ingredient, Recipe
+
+from recipes import serializers
+
+
+class IngredientViewSet(viewsets.GenericViewSet,
```

Comment:
Take into account you create this viewset but later you don't use it in tests, and it is not required by the application requirements.

Reply:
That's right!! I've just removed the unused viewset. Thanks :)